documentation/src/css/custom.css

```
@@ -1,95 +1,286 @@
-/* Performance Metrics */
+.hero__title {
+  font-size: 3.5rem;
+  font-weight: 900;
+  margin-bottom: 1rem;
+  background: var(--gradient-text);
+  -webkit-background-clip: text;
+  background-clip: text;
+  -webkit-text-fill-color: transparent;
+  text-shadow: 0 0 20px rgba(255, 0, 0, 0.5);
+  color: #f8f9fa !important;
+  text-shadow: 2px 2px 8px rgba(0, 0, 0, 0.6);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

text-shadow is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)